### PR TITLE
Update extractor docs

### PR DESCRIPTION
**Pull request by Michael-D-Johnson**
## Description
What: Change to the extractor documentation.
Details:  Extra documentation was added to original documentation.
- Links to GitHub 
- Steps of building an extractor
- Notes on debugging
- More advanced pyClowder examples

- Modified: /doc/src/sphinx/develop/extractor.rst. 
- Added: /doc/src/sphinx/_static/ug_extractors-*

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [ x] When possible

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have signed the CLA
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: doc/src/sphinx/develop/extractors.rst [hunk 1]
```diff
@@ -27,4 +34,268 @@ The current list of supported events is:
...
+
+.. code-block:: bash
+
+	2021-04-27 16:47:49,995 [MainThread     ] INFO
+
+
+Additional pyClowder Examples
+##############################
+
+For a simple example of an extractor, please refer to `extractor-csv <https://opensource.ncsa.illinois.edu/bitbucket/projects/CATS/repos/extractors-csv/browse>`_. This extractor is submitted on a csv file and returns the headers as metadata.
```
Comment (robkooper) [inline doc/src/sphinx/develop/extractors.rst:198]: use https://github.com/clowder-framework/extractors-csv

**Review by robkooper — CHANGES_REQUESTED** [review]
Also image 4 maybe red out the secret key.

File: doc/src/sphinx/develop/extractors.rst (hunk 1 above)
Comment (Michael-D-Johnson) [reply doc/src/sphinx/develop/extractors.rst]: I've committed a change for this

**Review by robkooper — APPROVED** [review]
need to make sure we squash the commits

File: doc/src/sphinx/develop/extractors.rst [hunk 2]
```diff
@@ -27,4 +34,268 @@ The current list of supported events is:
...
+
+11. You should see your metadata present if all worked successfully.
+
+
+A Quick Note on Debugging
+##########################
+
+To check the status of your extraction, navigate to the file within Clowder and click on the “Extractions” tab. This will give you a list of extractions that have been submitted. Any error messages will show up here if your extractor did not run successfully. 
+
+.. container:: imagepadding
```
Comment (lmarini) [inline doc/src/sphinx/develop/extractors.rst:155]: I had to remove the `.. container:: imagepadding` to get the images to show up.

Also the links all point to the same file `ug_extractors-1.png`.

I can make the changes but I wanted to check first that I wasn't missing 
something with the `.. container:: imagepadding`. Can I remove it? 

I am running sphinx-build 3.5.4

Thank you!

**Review by lmarini — CHANGES_REQUESTED** [review]

File: doc/src/sphinx/develop/extractors.rst (hunk 2 above)
Comment (Michael-D-Johnson) [reply doc/src/sphinx/develop/extractors.rst:155]: You can remove the imagepadding. I wasn't doing anything special with it. As for the images that have the prefix ug_extractors-, the order for the images as they appear in the doc is 1,2,4,3. Would you be able to make these changes alongside removing the imagepadding?

Comment (lmarini) [reply doc/src/sphinx/develop/extractors.rst:155]: Thank you. Yes I can make the changes.

**Pull request merged**